manipulate_pim_data/measurement/configure_measurement_limits.rst

```diff
@@ -0,0 +1,46 @@
+How to Configure Measurement Limits
+===================================
+
+In the Measurement Families, some limits about the number of families and units have been defined to guarantee that the PIM is functional and runs smoothly.
+However, as your catalog is unique, **you may need to raise those values**, this chapter explains how.
+
+.. warning::
+    If you hit those limits, this could reflect that **your modelization is not adapted**. If it's the case, we can't guarantee that the PIM will be functional and runs smoothly, be sure to test the performances with your new limits.
+
+All those limits are defined as parameters that you can override like any other `Symfony config parameter <https://symfony.com/doc/3.4/best_practices/configuration.html>`_.
```

Comment:
Wrong link, we are in SF 4.4

Reply:
You are right but this page does not exists anymore in 4.4 with all the changes on parameters. 
The closest we have in the docs is only targeted at applications, not changing parameters of bundles.
see https://symfony.com/doc/4.4/best_practices.html#use-parameters-for-application-configuration

Reply:
fixed